app/src/main/java/de/rochefort/childmonitor/audio/G711UCodec.kt

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2009 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Taken from https://android.googlesource.com/platform/external/nist-sip/+/6f95fdeab4481188b6260041b41d1db12b101266/src/com/android/sip/media/G711UCodec.java
+ *
+ */
+package de.rochefort.childmonitor.audio
+
+/**
+ * G.711 codec. This class provides u-law conversion.
+ */
+class G711UCodec {
+    fun decode(b16: ShortArray, ulaw: ByteArray, count: Int, offset: Int): Int {
+        var i = 0
+        var j = offset
+        while (i < count) {
+            b16[i] = table8to16[ulaw[j].toInt() and 0xFF]
+            i++
+            j++
+        }
+        return count
+    }
+
+    fun encode(b16: ShortArray, count: Int, b8: ByteArray, offset: Int): Int {
+        var i = 0
+        var j = offset
+        while (i < count) {
+            b8[j] = table13to8[b16[i].toInt() shr 4 and 0x1FFF]
+            i++
+            j++
+        }
+        return count
+    }
+
+    fun getSampleCount(frameSize: Int): Int {
```

Comment:
this was already unused before, lets get rid of it

Reply:
Will do.